ocaml/src/alba_osd.ml

```diff
@@ -136,7 +136,8 @@ class client
                    ~object_id_hint:None
                    ~fragment_cache:(alba_client # get_base_client # get_fragment_cache)
                    ~cache_on_write:(alba_client # get_base_client # get_cache_on_read_write |> snd)
-                 >>= fun (mf, _chunk_fidmos, _, gc_epoch) ->
+                   ~upload_slack:0.2
```

Comment:
upload_slack in this class should depend on what's configured in proxy config file too?